modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTypeSettings.java

```diff
@@ -94,9 +94,7 @@
 	@Override
 	public LocalizedValue predefinedValue();
 
-	@DDMFormField(
-		dataType = "date", type = "validation", visibilityExpression = "FALSE"
-	)
+	@DDMFormField(dataType = "date", label = "%validation", type = "validation")
```

Comment:
@brunofariasdeo please add a new commit called `Add feature flag` including the `visibilityExpression = "FALSE"`, since the functionality is not complete yet.